bin/update_cucumber

```diff
@@ -1,15 +1,23 @@
 #!/usr/bin/env php
 <?php
 
+/*
+ * This file is part of the Behat Gherkin Parser.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 $composerFile = __DIR__ . '/../composer.json';
-$composerConfig = file_get_contents($composerFile);
-foreach (json_decode($composerConfig, true, 512, JSON_THROW_ON_ERROR)['repositories'] as $repository)
-{
-    if ($repository['type'] !== 'package') {
+$composerConfigStr = file_get_contents($composerFile) ?: '';
+$composerConfig = (array) json_decode($composerConfigStr, true, 512, JSON_THROW_ON_ERROR);
+foreach ($composerConfig['repositories'] ?? [] as $repository) {
+    if (!isset($repository['type']) || $repository['type'] !== 'package') {
```

Comment:
Possibly slightly clearer and avoids having to keep the array keys in sync in future?
```suggestion
    if (($repository['type'] ?? null) !== 'package') {
```